deploy/ocp/manifests/0.13.0/0000_50_olm_03-clusterserviceversion.crd.yaml

```diff
@@ -6,7 +6,8 @@ metadata:
   name: clusterserviceversions.operators.coreos.com
```

Comment:
can we avoid editing already released manifests? just commit `/manifests` and the templates, and we can cut another release later?

Reply:
Sure thing.